text/068-coqide-split.md

```diff
@@ -0,0 +1,100 @@
+
+## Unresolved questions
+
+- Previous calls for volunteer maintainers (VsCoq1, Docker-Coq) have included a mention that the maintainers would be given recognition by adding them on the official Coq Team page. In this case, since the goal is to remove the official status of CoqIDE, it is not clear that this would be appropriate. If CoqIDE maintainers are acknowledged on this page, perhaps we should extend it to also acknowledge maintainers of other Coq IDEs, such as Proof General, Coqtail, and jsCoq.
+
+## Appendix: Call for volunteer maintainers of CoqIDE
+
+The Coq core team and Coq-community are looking for volunteer maintainers of the CoqIDE project.
+
+CoqIDE is an Integrated Development Environment (IDE) for Coq. CoqIDE is implemented using the OCaml programming language and the GTK3 widget toolkit for graphical user interfaces (GUIs). CoqIDE uses a legacy XML-based protocol to communicate with Coq and is licensed under the open source LGPL-2.1 license.
+
+CoqIDE's source code is currently part of Coq's GitHub repository. Due to a desire to shift IDE-related work toward LSP and VS Code support, the Coq core team no longer considers CoqIDE maintenance and evolution a priority. With this call, the team wants to give an opportunity to the Coq community to take over CoqIDE maintenance and lead its future evolution.
+
+If maintainers are found, CoqIDE sources will be moved to a repository in the Coq-community organization on GitHub, where it will receive support and fixes from the core team for a limited time. If no maintainers are found, CoqIDE sources will be kept in Coq's repository and minimally maintained until their eventual removal.
+
+More details about the context and the plans for the future of IDEs for Coq can be found in the CEP leading to this call: \<add link to this CEP here\>
+
+Please respond to this GitHub issue with a brief motivation and summary of relevant experience for becoming a CoqIDE maintainer. As part of their application, volunteer maintainers are encouraged to briefly present their short-term and long-term plans for CoqIDE and how long they think they will remain active on CoqIDE maintenance. However, this won't be considered as a commitment on their part, as plans and priorities can evolve based on the context and personal circumstances. The maintainer(s) will be selected from the issue responders by the Coq core team and Coq-community organization owners. Responders not selected will still be encouraged to contribute to CoqIDE in collaboration with the new maintainer(s) and other contributors.
```

Comment:
How about reformatting the entire document so all the lines are of a reasonable length, e.g. 80 characters max?

Reply:
And adding the link to this CEP? `<add link to this CEP here>`

Reply:
We will do two versions of the call for maintainers, one for Coq Discourse (Markdown) and one for email (Coq-club). The formatting of these calls will be different, so I don't see any point of pre-formatting the CEP appendix right now.

Reply:
> How about reformatting the entire document so all the lines are of a reasonable length, e.g. 80 characters max?

You should probably read the CEP using the Markdown renderer (e.g., the link in the top post).

> And adding the link to this CEP? `<add link to this CEP here>`

This link should be a link to the merged CEP, so the document rather than the pull request, and it would be a broken link until we merge it.

Reply:
> I don't see any point of pre-formatting the CEP appendix right now.

I suggested reformatting the entire document, not just the appendix